### ISSUE TITLE
When fullbleed panels need to scroll vertically, two scrollbars show up

### ISSUE THREAD

**Issue by Lange** [nodecg/nodecg#312]
Dunno why this happens. The outer scrollbar is exactly 3px. I've inspected every DOM element in the hierarchy and they are all precisely the height they should be, yet this 3px scroll remains. I don't have time to put together an example bundle that demonstrates this problem, getting on a plane in a few hours. Would really appreciate any help here.

**Comment by Lange** [comment]
I am adding a $150 bounty to this issue. If I merge a pull request from you that resolves this issue, the bounty is yours.

Here is a gfycat that clearly illustrates the problem: https://gfycat.com/WholeFlickeringGalapagoshawk

[![image](https://user-images.githubusercontent.com/873012/29006329-299ffaac-7ab3-11e7-8106-2026a354b271.png)](https://gfycat.com/WholeFlickeringGalapagoshawk)

Outermost scrollbar that moves only 3px up and down is the one that shouldn't be there, and that I have been trying to get rid of.

I have also created a small test bundle and an installer script to get you up and running as fast as possible, so that you can focus on debugging and investigation, not fumbling with setting up my weird code. **You must also be on Node.js 7 or greater (8 is recommended)**:

```bash
#!/bin/bash

# Install tools used during setup.
npm install -g bower

# Clone NodeCG and checkout the 0.9-dev branch.
git clone https://github.com/nodecg/nodecg.git
cd nodecg
git checkout 0.9-dev

# Install NodeCG's npm and bower dependencies.
npm install --production
bower install

# Install the test bundle, which convinently illustrates this issue.
cd bundles
curl -o workspace-test.zip https://dl.dropboxusercontent.com/u/6089084/workspace-test.zip
unzip workspace-test.zip
rm workspace-test.zip

# Start NodeCG.
cd ..
node index.js
```

Once you have NodeCG running, open http://localhost:9090/dashboard/#fullbleed/fullbleed-panel to see a sample fullbleed workspace which reproduces this issue.

**Comment by tylerthardy** [comment]
I've got a fix, but I'm not familiar with your build environment. Can I do a pull request where I've modified both the src and the build manually?

**Comment by Lange** [comment]
@tylerthardy It looks like it's already been fixed by https://github.com/nodecg/nodecg/pull/316

Was your fix something different?

**Comment by tylerthardy** [comment]
Drat, that was it. Totally missed the fix.

**Comment by Lange** [comment]
Fixed by @Seldszar's #316 pull request. The PR has been merged and the bounty has been paid out.
